docs/WIP/Tutorial_Motif_only_Matrix_Profile.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        min_indices = (0, 0)
</pre><p>This creates an immutable tuple object every time (and also later when you set it <code>(i, j)</code> . It might make sense to do <code>min_indices = np.array([0, 0])</code> and then, later, simply do <code>min_indices[:] = i, j</code></p><p> </p><p>If it is undesirable to return an <code>np.array</code> then simply do <code>min_i = 0</code> and <code>min_j = 0</code> and then replace with <code>min_i, min_j = i, j</code> and then <code>return  bMPdsr, (min_distance, (min_i, min_j))</code> at the end</p><p> </p><p>In fact, why do we need to return a tuple? Why not just return <code>bMPdsr, min_distance, min_i, min_j</code> ? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>